test/src/dao/EnabledAwareDAO.js

```diff
@@ -4,7 +4,7 @@
  * http://www.apache.org/licenses/LICENSE-2.0
  */
 
-fdescribe('EnabledAwareDAO', function() {
+describe('EnabledAwareDAO', function() {
```

Comment:
Just want to make sure I understand what's happening here. Was this `fdescribe` causing other tests not to execute?

Reply:
Yes and the CI only runs this describe for the quite some times. We were trying to fix it last time but the object cloning was in the way #1812.